.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+> Please fill out this template when filing an issue.
+> All lines marked as quotes (`>`) instruct you with
+> what info we expect.
+> Remove this opening paragraph.
+
+## Goals
+
+> Replace this with what you tried to achieve.
+
+## Expected Results
+
+> Replace this with what you expected to happen.
+
+## Actual Results
+
+> Replace this with what actually happened.
+> e.g. the full stack trace with exception messages.
+
+## Steps to Reproduce
+
+> Provide steps we can follow to reproduce this issue.
+
+## Code Sample
+
+> Provide a code sample that highlights the issue.
+> If relevant, include your model definitions.
+> For larger code samples, links to external gists or repositories are preferred.
+> Full Android Studio projects that we can compile ourselves are ideal.
+> See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code
+> for how to do code formatting.
+
+## Version of Realm and Tooling
+
+> Share which versions of Realm are affected, the Android Studio version
```

Comment:
I see your point, and yes, I guess the most errors we see related to buildtools are Instant Run related anyway, so Android Studio/Android version/device is probably fine.